core/modules/layout/templates/layout--flexible.tpl.php

```diff
@@ -42,7 +42,7 @@
         <?php endif; ?>
         <div class="l-flexible-row row">
         <?php foreach ($row['regions'] as $region): ?>
-          <div class="l-col col-md-<?php print $region['region_md']; ?>">
+          <div class="l-col col-md-<?php print $region['region_md']; ?> <?php print backdrop_html_class($region['region_name']); ?>">
```

Comment:
Should we be prefixing the class with `region-` for consistency?

```suggestion
          <div class="l-col col-md-<?php print $region['region_md']; ?> region-<?php print backdrop_html_class($region['region_name']); ?>">
```